core/src/runtime/RuntimeClient.ts

```
@@ -192,6 +192,24 @@ export class RuntimeClient {
         return runtimeHcAgentInfos
     }
 
+    async getNetworkMetrics(): Promise<String> {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)